data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1918 @@
+			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the Remote Blue operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but trouble you with that, I will not. Very useful to us your support has been, Captain."`
+
+mission "Heliarch Containment 5"
+	name "Stop The Fleeing Ships"
+	description "Defeat the criminal ships that are attempting to flee. You must disable and board their ships, in order to help the Heliarchs find the source of their weapons."
+	landing
+	to offer
+		has "Heliarch Containment 4-B: done"
+		not "Resistance: Questions: done"
+	source "Station Cian"
+	on offer
+		"reputation: Lunarium" = -1000
+		event "resistance fleeing rb"
+		conversation
+			`After the transmission with Plortup ends, the station is awfully quiet for a few hours. You head outside your ship to see if you can find something interesting to do, but you don't find anything particularly interesting, as the station holds few shops. You move to the job board, but find it completely empty. You also notice that the market's trade values are not updating, and any news system is non-functional.`
+			`	Suddenly, some of the few Heliarchs still stationed here rush past you heading for an office. They quickly reemerge and approach you.`
+			`	"Captain! Helped us, you have, yes? Ainding in the Remote Blue operation?" You nod. "Think you trustworthy, the consul does, and in dire need of a combat-ready ship, we are. Aid us, you must!"`
+			choice
+				`	"What's going on?"`
+				`	"My ship isn't exactly combat-ready to fight your own warships."`
+			`	They explain that it's not simply the station's mercantile and news systems that haven't been functioning properly - they haven't been able to contact the group on Remote Blue, nor the fleet that left for Fourth Shadow, nor get in any sort of contact with anywhere else in the Coalition.`
+			`	"Only readings of the local systems, we can get, and alerted us, they have, of incoming ships! Armed, civilian ships! Criminals, no doubt, and fleeing Remote Blue, they are!"`
+			`	You can't say much before they've practically pushed you back into your ship. They promise you'll be greatly compensated for this, and ask that you only disable the ships so you can board and check for hostages, or possibly allow for capture and questioning of the outlaws.`
+				launch
+	npc board
+		personality timid plunders entering target
+		government "Lunarium"
+		ship "Saryd Sojourner (Bombardments)" "Hodoviah"
+		conversation
+			`Unlike the other two vessels, this ship is equipped with Heliarch weaponry. Unfortunately you can neither verify the equipment it is using nor check for hostages, as a self destruction sequence starts when you're about to enter it.`
```

Comment:
The player might not know what the other two ships are equipped with.

Reply:
Changed it to not expect the other two were boarded first.